engine/src/cmd/movable.h

```diff
@@ -0,0 +1,197 @@
+#ifndef MOVABLE_H
+#define MOVABLE_H
+
+#include "gfx/vec.h"
+#include "vs_limits.h"
+#include "gfx/quaternion.h"
+
+class Transformation;
```

Comment:
Should be `struct` not `class`, to match its definition elsewhere

Reply:
do we even need the forward declaration? If not, just drop it. Otherwise yes it needs to match up.

Reply:
done.